src/hotspot/share/interpreter/templateInterpreterGenerator.cpp

```diff
@@ -228,6 +228,7 @@ void TemplateInterpreterGenerator::generate_all() {
   // entries for `native` methods to use the same address in case
   // intrinsic is disabled.
   native_method_entry(java_lang_Thread_currentThread)
+  native_method_entry(java_lang_ref_reference_get0)
```

Comment:
It turned out there was a bug lurking in the change to move the intrinsic to
Reference::get0. I had tested it with the interpreter intrinsic made
inoperative, but nearly forgot to test the normal case. It turned out that if
the interpreter intrinsic was operational but disabled then the interpreter
would hit an assert "tried to execute native method as non-native". This line
is the fix for that.